package.json

```diff
@@ -31,9 +31,13 @@
     "@wordpress/html-entities": "2.10.0",
     "@wordpress/i18n": "3.17.0",
     "@wordpress/icons": "2.9.0",
+    "@wordpress/is-shallow-equal": "3.0.0",
     "@wordpress/url": "2.21.0",
     "classnames": "2.2.6",
+    "clipboard": "2.0.6",
     "prop-types": "15.7.2",
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
```

Comment:
It would be best to keep `react`, `react-dom` and `react-test-renderer` and the same versions to prevent any JS build or test errors.